helm/releases-aws/templates/releases.yaml

```diff
@@ -0,0 +1 @@
+../../../aws.yaml
```

Comment:
Why is this needed?

Reply:
Just for my little brain, what kind of problem does this solve? What does not work without that?

Reply:
It feels unwieldy to ask everyone to edit `helm/releases-aws-templates/releases.yaml` when creating a release (deployed to CPs in a helm chart). Using a symlink we can keep `aws.yaml` in the root of the repo as it has been and have it reflected in the helm chart without any copying magic in CircleCI. This is seamless when interacting with it on your local filesystem, but it does look strange as presented here on GitHub.

Reply:
I see. That is nice. Thanks!